static/openapi/dem/elevation-openapi.yaml

```diff
@@ -0,0 +1,131 @@
+      requestBody:
+        description: Points to calculate heights for.
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/pointsReq'
+          application/octet-stream:
+            schema:
+              type: string
+              format: binary
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/pointsRes'
+            application/octet-stream:
+              schema:
+                type: string
+                format: binary
+        '400':
+          description: Points density is too low to compute
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/error'
+        '500':
+          description: Service Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/error'
+components:
+  schemas:
+    error:
+      type: object
+      required:
+        - message
+        - errorCode
+      properties:
+        message:
+          type: string
+        errorCode:
+          oneOf:
+            - $ref: '#/components/schemas/errorCodesEnum'
+    pointsReq:
+      type: object
+      required:
+        - positions
+      properties:
+        positions:
+          type: array
+          items:
+            type: object
+            required:
+              - longitude
+              - latitude
+            properties:
+              longitude:
+                type: number
+                format: double
+              latitude:
+                type: number
+                format: double
+        productType:
+          oneOf:
+            - $ref: '#/components/schemas/productTypeEnum'
+          default: MIXED
+        excludeFields:
+          type: array
+          uniqueItems: true
+          items:
+            oneOf:
+              - $ref: '#/components/schemas/excludeFieldsEnum'
+    pointsRes:
+      type: object
+      required:
+        - data
+      properties:
+        data:
+          type: array
+          items:
+            type: object
+            required:
+              - longitude
+              - latitude
+              - height
+            properties:
+              longitude:
+                type: number
+                format: double
+              latitude:
+                type: number
+                format: double
+              height:
+                nullable: true
+                type: number
+                format: double
+    productTypeEnum:
+      type: string
+      enum:
+        - DSM
+        - DTM
+        - MIXED
+    excludeFieldsEnum:
+      type: string
+      enum:
+        - productType
+        - updateDate
+        - resolutionMeter
+    errorCodesEnum:
+      type: string
+      enum:
+        - POINTS_DENSITY_TOO_LOW_ERROR
+        - GENERAL_SERVER_ERROR
```

Comment:
Please add a line at the end of the file.

Reply:
DONE